dev-assets/index.html

```diff
@@ -28,6 +28,17 @@
     I'm a <em>native</em> child
   </my-component>
 
+  <my-component2
+    id='"myComponent2"'
```

Comment:
no need to do this, when we use `try ... catch` around `JSON.parse` with fallback to orig. value (string)